### ISSUE TITLE
Crash after refreshing a renamed folder

### ISSUE THREAD

**Issue by ynynl** [UrosVuj/Explorer-Bookmark#10]
The extension crashed and lost all the bookmarks after clicking the refreshing button.
reproduce step:
1. bookmark a folder
2. rename the folder
3. click refreshing button
4. see the error
```
Error: ENOENT: no such file or directory, stat 'the/renamed/folder/path'
```

**Comment by zhanghm1995** [comment]
@ynynl Same as this error, have you solved this problem?

**Comment by UrosVuj** [comment]
This issue was fixed in version 1.0.1

It was caused by a dependency that had a breaking change.
